### PR TITLE
simple msitools installer

### PR DESCRIPTION
**Pull request by tillkrischer**
This is a installer using msitools.
Msitools is limited in functionality: It doesn't support any GUI.
This means that the user is unable to pick the target folder, which is not ideal.
But it also means that the app will always be installed to `C:\Program Files (x86)\Browserpass`.
So in this installer the path is just hard-coded. 

```
make configure-windows
make browserpass-windows64
make setup.msi
```

About WSL:
I don't think you can put files into the WSL filesystem from a windows installer,
so installing browserpass in WSL has to be a separate step. 
But you could make a version of this installer that puts `browserpass-wsl.bat` in place of ` browserpass-windows64.exe`

### CODE REVIEW

File: setup.wxs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    <Directory Id='TARGETDIR' Name='SourceDir'>
+      <Directory Id='ProgramFilesFolder' Name='PFiles'>
+        <Directory Id='INSTALLDIR' Name='Browserpass'>
+          <Component Id='MainExecutable' Guid='ABCDDCBA-83F1-4F22-985B-FDB3C8ABD471'>
+            <File Id='BrowserpassEXE' Name='browserpass-windows64.exe' Source='browserpass-windows64.exe' KeyPath='yes'/>
+            <File Id='ChromiumHostJSON' Name='chromium-host.json' Source='browser-files/chromium-host.json' />
+            <File Id='ChromiumPolicyJSON' Name='chromium-policy.json' Source='browser-files/chromium-policy.json' />
+            <File Id='FirefoxHostJSON' Name='firefox-host.json' Source='browser-files/firefox-host.json' />
+            <RegistryKey Id='ChromeNativeMessaging' Root='HKCU' Key='Software\Google\Chrome\NativeMessagingHosts\com.github.browserpass.native' >
```
Comment (maximbaz) [inline setup.wxs]: Given that we install in a "global" folder, let's also configure browsers in `HKLM` to have effect for all users.

Comment (maximbaz) [reply setup.wxs]: actually, I shouldn't have suggested to configure _both_, having only `HKLM` would probably be enough 🤔

**Comment by maximbaz** [comment]
I noticed you bumped `InstallerVersion='200'`, is that something I need to bump every time I make new releases? Every time I make changes to the `.wxs` file? Or it doesn't really impact anything?

**Comment by tillkrischer** [comment]
> I noticed you bumped `InstallerVersion='200'`, is that something I need to bump every time I make new releases? Every time I make changes to the `.wxs` file? Or it doesn't really impact anything?

When changed it to 64bit I was getting a warning, that 64bit apps need an InstallerVersion of at least 200. (Not sure what that even means).
You don't have to increment it further I think.

**Comment by maximbaz** [comment]
Gotcha, thank you! Finally having a proper Windows installer is HUGE!!

I want to rearrange some bits and update docs, but I will do it myself in a serapate PR, thank you so much for making it happen!

**Comment by tillkrischer** [comment]
No Problem, I am happy to help out.

**Pull request merged**